sdk/js/src/Request.ts

```diff
@@ -40,16 +41,16 @@ export const get = async (url: string): Promise<AxiosResponse> => {
     })
 }
 
-export const getConfigJson = async (
-    environmentKey: string, 
-    user: DVCPopulatedUser, 
-    enableCloudData?: boolean
-): Promise<BucketedUserConfig> => {
-    const queryParams = `${serializeUser(user)}${enableCloudData ? ('&enableCloudEntityData=' + enableCloudData): ''}`
-    const url = `${BASE_URL}${HOST}${CONFIG_PATH}?envKey=${environmentKey}${queryParams && '&' + queryParams}`
+export const getConfigJson = async (environmentKey: string, user: DVCPopulatedUser, enableCloudData?: boolean): Promise<BucketedUserConfig> => {
+    const queryParams = new URLSearchParams(`${serializeUser(user)}`)
+    queryParams.append('envKey', environmentKey)
+    if (enableCloudData) {
+        queryParams.append('enableCloudEntityData', 'true')
+    }
+    const url = new URL(`${BASE_URL}${HOST}${CONFIG_PATH}?${queryParams.toString()}`)
```

Comment:
you can also set `url.searchParams` to the URLSearchParams object I think

Reply:
it's a read only property apparently